src/interfaces/http/router.js

```diff
@@ -45,14 +45,15 @@ module.exports = ({ config, logger, database }) => {
    * The `controllerPath` is relative to the `interfaces/http` folder
    */
 
-  apiRouter.use('/', controller('index'))
-  apiRouter.use('/token', controller('token').router)
-  apiRouter.use('/users', controller('user').router)
-  apiRouter.use('/companies', controller('company').router)
+  apiRouter.use("/", controller("index"));
+  apiRouter.use("/token", controller("token").router);
+  apiRouter.use("/users", controller("user").router);
+  apiRouter.use("/companies", controller("company").router);
+  /* PLOP_INJECT_ROUTE */
 
-  router.use(`/api/${config.version}`, apiRouter)
+  router.use(`/api/${config.version}`, apiRouter);
 
-  router.use(partialRight(errorHandler, [logger, config]))
+  router.use(partialRight(errorHandler, [logger, config]));
 
-  return router
-}
+  return router;
+};
```

Comment:
I think you have used `prettier` here instead of `standard`?

Reply:
yes, it's the default throughout our VS Code setup

Reply:
I was actually planning to move this also to `prettier` because most of developers are working with that in their frontend code. 

Created an [issue here](https://github.com/joshuaalpuerto/node-ddd-boilerplate/issues/70). Let have that in separate PR.